src/LandingScreen.js

```diff
@@ -8,23 +8,30 @@ export default class LandingScreen extends Component {
     super();
   }
 
+  displayTrailsByLocation = () => {
```

Comment:
we might need to rename this method to something like "updateTrailsByLocation" since it's not actually doing any displaying.

Reply:
⬆️